.github/workflows/ci.yml

```diff
@@ -42,7 +42,10 @@ jobs:
           args: -- --check
 
   clippy:
-    name: Clippy
+    name: Clippy ${{ matrix.args }}
+    strategy:
+      matrix:
+        args: ["", "--tests"]
```

Comment:
@CAGS295 I'm not sure if we need a matrix here. Maybe `--all-tagets` is a better option? What do you think?

Reply:
I prefer `--all-targets --all-features`